src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -171,6 +171,31 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+[Dell NFV Settings]
```

Comment:
There is a second Dell NFV Settings section below at https://github.com/dsp-jetpack/JetPack/pull/28/files#diff-03907ea6ba7811b5a8d4acb7b8c60701L255.
One of these sections needs to be removed.

Reply:
Removed second one. Kept only one as suggested.